core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt

```diff
@@ -9,4 +9,5 @@ public actual interface SubscriptionOption {
     public actual val presentedOfferingIdentifier: String?
     public actual val presentedOfferingContext: PresentedOfferingContext?
     public actual val purchasingData: PurchasingData
+    public actual val installmentsInfo: InstallmentsInfo?
```

Comment:
I've added this as an optional for iOS when I know it'll always be null. Do I need to add anything to make sure it's being correctly parsed as `null` when it's being parsed out from PHC?

Reply:
The `SubscriptionOption` object itself should always be null in iOS, so I don't think we need to do any extra parsing.

Reply:
I just tested making a purchase through a paywall in iOS through the KMP purchase tester app and everything worked correctly, so I think we're probably good here.